app/models/node.ts

```diff
@@ -130,25 +130,25 @@ export default OsfModel.extend(FileItemMixin, {
      * @param {String} userId
      * @return {boolean} Whether the specified user is a contributor on this node
      */
-    isContributor(userId) {
+    isContributor(userId: string): Promise<any> | Promise<boolean> {
```

Comment:
This may as well be just `Promise<any>`. The union with `Promise<boolean>` doesn't add anything.

It looks like it does always resolve to a boolean, though, could we use just `Promise<boolean>`?

(Another smelly thing that might be too much to ask of you: I dislike that this method is named like it returns boolean, but actually returns a promise...)